docs/operate-a-stake-pool/hardware-requirements.md

```diff
@@ -9,11 +9,11 @@ The latest technical specifications and supported platforms can be found on the
 
 Currently the following specifications are recommended:
 
-- Servers: 1 for block producer node, at least 1 for relay node
+- Servers: 1 for block producer node, at least 1 as relay node
 - CPU: An Intel or AMD x86 processor with two or more cores, at 1.6GHz or faster (2GHz or faster for a stake pool or relay)
-- Memory: 16 GB of RAM
-- Storage: 100GB of free storage (150GB recommended for future growth)
-- Operating system: 64-bit Linux (i.e. Ubuntu Server 20.04 LTS)
+- Memory: 24GB of RAM
+- Storage: 150GB of free storage (250GB recommended for future growth)
+- Operating system: 64-bit Linux (Ubuntu 18.04+, Mint 19.3+, Debian 10.3+), Windows 8.1+, MacOS 10.13+
```

Comment:
If you check supported roles, BP and Relay are  not "Supported" as per release notes for Windows/Mac - thus, IMO maybe good to keep eliminated in docs for "stake pool operators"?
![IMG_20230729_224429](https://github.com/cardano-foundation/developer-portal/assets/3169068/daef15c6-0875-4c71-a08c-c901d73c8170)

Reply:
You know, I recalled seeing that many times but couldn't find it today... right under the accordion on the node release page itself. 😖 Yes I've removed the Mac & Windows roles (again).